src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -125,11 +125,4 @@ private void showNoPerson(Model model) {
 
         assertTrue(model.getFilteredPersonList().isEmpty());
     }
-
```

Comment:
Sorry could you explain why showAllPersons is moved to CommandTestUtil but not showNoPerson

Reply:
Trying to keep the methods to the smallest scope possible! `showNoPerson` is only used in `DeleteCommandTest`, and doesn't seem like it is going to be used anywhere else since no other commands expect deletion or changes that will change the displayed view to no person (find command can directly update the display using the predicate).

`showAllPersons` has started to be used in `RemarkCommandTest` in this PR, so I moved it to CommandTestUtil. LMK if you think `showNoPerson` should also be moved, otherwise this can be merged! 👍

Reply:
thanks for the explanation! ill merge it now :)